CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Support for `ipfs`/`ipns` URLs
 - Mode field for regex hint bindings
 
+### Changed
+
+- `ExpandSelection` is now a configurable mouse binding action
```

Comment:
Would you prefer a longer explanation indicating that users will need to add this binding if they have custom mouse bindings and want to keep this behaviour?

Reply:
This should behave identically to other default key bindings, being enabled by default even if other bindings are defined.